mycore-orcid2/src/main/resources/xsl/orcid2/v3/mcr2work_generic.xsl

```diff
@@ -54,14 +56,16 @@
     </common:subtitle>
   </xsl:template>
 
-  <xsl:template match="mods:mods/mods:titleInfo[@type='translated']">
+  <xsl:template match="mods:mods/mods:titleInfo[@type='translated'][1]">
```

Comment:
here we could perhaps prefer english or german or application configured languages instead of first translation, but as bugfix this is ok for me